src/main/java/org/cactoos/text/ComparableText.java

```diff
@@ -69,4 +70,24 @@ public String asString() throws Exception {
         return this.text.asString();
     }
 
+    @Override
+    public boolean equals(final Object other) {
+        final boolean result;
+        if (this == other) {
+            result = true;
+        } else if (other == null || this.getClass() != other.getClass()) {
+            result = false;
+        } else {
+            result = Objects.equals(
+                this.text,
+                ((ComparableText) other).text
+            );
+        }
+        return result;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(this.text);
```

Comment:
@ilyakharlamov Let's avoid use static methods.

Reply:
@fabriciofx wait for #978 I guess

Reply:
@fabriciofx fixed in 93445f16